data/mods/Magiclysm/effects/effects.json

```diff
@@ -475,6 +475,57 @@
       }
     ]
   },
+  {
+    "id": "electric_eye",
+    "type": "effect_type",
+    "name": [ "Electric Eye" ],
+    "desc": [
+      "Enhance your sight with your powers.  It allows you to see creatures of the same energy around you, and some minor bonuses."
+    ],
+    "apply_message": "Your sight is enhanced.",
+    "remove_message": "Your sight turn back to usual state.",
+    "rating": "good",
+    "show_intensity": false,
+    "enchantments": [
+      {
+        "values": [
+          { "value": "BONUS_BLOCK", "add": 1 },
+          { "value": "BONUS_DODGE", "add": 1 },
+          { "value": "DEXTERITY", "add": 2 },
+          { "value": "PERCEPTION", "add": 4 },
+          { "value": "SIGHT_RANGE_ELECTRIC", "add": 1000 }
+        ]
+      }
+    ]
+  },
+  {
+    "id": "electric_waves",
+    "type": "effect_type",
+    "name": [ "Electric Waves" ],
+    "desc": [ "Your body throw a massive electric charges when you attack monsters." ],
+    "apply_message": "A powerful energy streams out of your hands.",
+    "remove_message": "Energy streams calm down.",
+    "rating": "good",
+    "show_intensity": false,
+    "//": "When #61693 will be resolved (or, in other words, the spell will start to throw electric waves, as it says), the ITEM_DAMAGE_ELEC must be removed, and the spell description must be edited.  Artenatively, the effect may be moved into electric gloves, instead of their flat bonus.",
```

Comment:
```suggestion
    "//": "When #61693 will be resolved (or, in other words, the spell will start to throw electric waves, as it says), the ITEM_DAMAGE_ELEC must be removed, and the spell description must be edited.  Alternatively, the effect may be moved into electric gloves, instead of their flat bonus.",
```